src/test/resources/data/InstructorStudentListPageUiTest.json

```diff
@@ -144,10 +144,10 @@
     },
      "Student3Course2": {
       "googleId": "",
-      "email": "barney.c.tmms@gmail.tmt",
+      "email": "hugh.c.tmms@gmail.tmt",
```

Comment:
They should also follow the same format, maybe "Ivan James"(ivan.j) and "Jack Khan"(jack.k)?